main.tf

```
@@ -155,10 +138,10 @@ resource "aws_security_group_rule" "allow_all_egress" {
 
 module "kms_key" {
   source  = "cloudposse/kms-key/aws"
-  version = "0.10.0"
+  version = "0.12.1"
 
-  enabled = var.session_logging_enabled && var.session_logging_encryption_enabled && var.session_logging_kms_key_arn == ""
-  context = module.logs_label.context
+  enabled = var.session_logging_enabled && var.session_logging_encryption_enabled && length(var.session_logging_kms_key_arn) == 0
+  context = module.this.context
```

Because we're keeping the same context EXCEPT the `attributes`, we'd break existing users usage of this module when they upgrade. Let's add that attributes, so we don't mess up folks usage of this module. 

```suggestion
  context = module.this.context
  attributes = attributes = compact(concat(["logs"], module.this.attributes))
```